genesyscloud/integration_credential/resource_genesyscloud_integration_credential.go

```diff
@@ -54,24 +52,6 @@ func getAllCredentials(ctx context.Context, clientConfig *platformclientv2.Confi
 	for _, cred := range *credentials {
 		log.Printf("Dealing with credential id : %s", *cred.Id)
 		if cred.Name != nil { // Credential is possible to have no name
-
```

Comment:
William originally added this code because we were trying to leverage the naming standards for integrations to go figure out the integration credential associated with the integration.  However, after chatting with the integration team they let us know that this is not possible.  So we need to remove the code below because it does not add value.